packages/components/src/tools-panel/stories/utils/tools-panel-with-item-group-slot.tsx

```diff
@@ -90,49 +97,83 @@ const SlotContainer = ( { Slot: ToolsPanelSlot, ...props } ) => {
 
 // This wraps the slot with a `ToolsPanel` mimicking a real-world use case from
 // the block editor.
-ToolsPanelItems.Slot = ( { resetAll, ...props } ) => (
+ToolsPanelItems.Slot = ( {
+	resetAll,
+	panelId,
+	label,
+	hasInnerWrapper,
+	shouldRenderPlaceholderItems,
+	__experimentalFirstVisibleItemClass,
+	__experimentalLastVisibleItemClass,
+	...props
+}: React.ComponentProps< typeof ToolsPanel > & {
+	[ key: string ]: any;
+} ) => (
 	<ToolsPanel
-		label="Tools Panel with Item Group"
+		label={ label }
 		resetAll={ resetAll }
 		panelId={ panelId }
-		hasInnerWrapper={ true }
-		shouldRenderPlaceholderItems={ true }
-		__experimentalFirstVisibleItemClass="first"
-		__experimentalLastVisibleItemClass="last"
```

Comment:
Moved these from being hardcoded values, to being parameters (passed via `ToolsPanelWithItemGroupSlot.args`).

I then had to add prop types for `ToolsPanelItems.Slot`, although this is still quite messy due to the mixing of `ToolsPanel`-specific props with other generic (`any`) props that get forwarded to `Slot`